projects/1.18/assets/immersive-engineering/immersiveengineering/lang/zh_cn.json

```
@@ -496,6 +510,7 @@
   "subtitle.immersiveengineering.direSwitch": "Direwolf20正在模仿刀闸",
   "subtitle.immersiveengineering.skyhook": "天钩沿线缆滑行",
   "subtitle.immersiveengineering.chute": "物品从槽中滑过",
+  "subtitle.immersiveengineering.glider": "滑翔翼受损",

```

```suggestion
  "subtitle.immersiveengineering.spray": "流体：溅射",
  "subtitle.immersiveengineering.sprayFire": "燃烧中的流体：溅射",
  "subtitle.immersiveengineering.chargeFast": "电容器：充电",
  "subtitle.immersiveengineering.chargeSlow": "电容器：充电",
  "subtitle.immersiveengineering.spark": "电火花",
  "subtitle.immersiveengineering.railgunFire": "磁轨炮：发射",
  "subtitle.immersiveengineering.tesla": "特斯拉线圈：电击",
  "subtitle.immersiveengineering.direSwitch": "Direwolf20正在模仿刀闸",
  "subtitle.immersiveengineering.skyhook": "天钩：沿线缆滑行",
  "subtitle.immersiveengineering.chute": "物品：从槽中滑过",
  "subtitle.immersiveengineering.glider": "滑翔翼：受损",
```
~~全部瞎改~~，可以对照一下原版鞘翅的字幕（如果有的话）